src/js/view/controls/controlbar.js

```diff
@@ -120,6 +120,8 @@ export default class Controlbar {
         const localization = _model.get('localization');
         const timeSlider = new TimeSlider(_model, _api, _accessibilityContainer.querySelector('.jw-time-update'));
         let volumeTooltip;
+        let volumeTooltipEl;
+        let muteTip;
```

Comment:
If `volumeTooltipEl` are only used in the `if (!this._isMobile)` block below they should be defined as constants in that block.